.github/dod.yaml

```diff
@@ -5,3 +5,7 @@ dod:
   - 'Acceptance criteria of the linked stories satisfied'
   - 'Code change comprehended by those who approved it'
   - 'State of target branch maintained or improved after the code change'
+  - 'Go to MainActivityTest in the Android App and uncommit @Test'
+  - 'Run test manually '
+  - 'Add a picture of the test result in the comments'
+  - 'Comment @Test again to avoid future problems with the CI' 
```

Comment:
These will take effect starting from the next PR.
However, it feels like that all these points should be once instead?

Reply:
Maybe you can try adding `\n` in the text and see how it will be rendered in the end. Unfortunately you need to merge first to see the results, but doesn't hurt to try. I can approve things quickly if need be.

Reply:
![image](https://user-images.githubusercontent.com/90027419/168644078-22b02035-804b-4659-8558-596a0e9dd295.png)
Like this?

Reply:
Yeap :+1:

Reply:
could you please merge this pull request ? =)